lib/boat.rb

```diff
@@ -0,0 +1,16 @@
+class Boat
+  attr_reader :type, :price_per_hour, :total_hours
+  def initialize(type, price_per_hour)
+    @type = type
+    @price_per_hour = price_per_hour
+    @total_hours
+  end
+
+  def hours_rented(time)
```

Comment:
This is a dangerous method - it would allow anyone to overwrite the instance variable `@total_hours` without going through the `add_hours` method.  You should get rid of this method and just rely on `add_hours` and `@total_hours`